### PR TITLE
remove data wrapper from body and Standardize functions

### PR DESCRIPTION
**Pull request by felipeweb**
related issue #145 and #142

### CODE REVIEW

File: adapters/postgres/postgres.go [hunk 1]
```diff
@@ -133,6 +132,67 @@ func WhereByRequest(r *http.Request, initialPlaceholderID int) (whereSyntax stri
...
+		}
+
+		initialPlaceholderID++
+	}
+	setSyntax = strings.Join(fields, ", ")
+	return
+}
+
+// InsertByRequest create insert SQL
+func InsertByRequest(r *http.Request) (colsName string, colsValue string, values []interface{}, err error) {
```
Comment (marioidival) [inline adapters/postgres/postgres.go]: Now, pREST have more then one way of insert things? This function name isn't good.

File: adapters/postgres/postgres.go [hunk 2]
```diff
@@ -353,36 +413,15 @@ func parseArray(value interface{}) string {
 }
 
 // Insert execute insert sql into a table
-func Insert(database, schema, table string, body api.Request) (jsonData []byte, err error) {
+func Insert(database, schema, table string, r *http.Request) (jsonData []byte, err error) {
```
Comment (marioidival) [inline adapters/postgres/postgres.go]: It's better to others functions (like UPDATE), don't receive `r *http.Request`. I prefer send the body just decoded...
```go
body := make(map[string]interface{})
if err = json.NewDecoder(r.Body).Decode(&body); err != nil {
        return
}```
This snippet in controller.

File: adapters/postgres/postgres.go (hunk 1 above)
Comment (felipeweb) [inline adapters/postgres/postgres.go]: @marioidival no, we will break the compatibility in next release. this PR is first step to turn pREST more easy to  as a package

File: adapters/postgres/postgres.go (hunk 2 above)
Comment (felipeweb) [inline adapters/postgres/postgres.go]: @marioidival  i'll do it in next PR when will solve https://github.com/nuveo/prest/issues/142

File: adapters/postgres/postgres.go (hunk 1 above)
Comment (marioidival) [inline adapters/postgres/postgres.go]: Ok, but function name isn't good yet

**Pull request merged**